README.md

```diff
@@ -149,3 +149,12 @@ In order to create an instance of gatekeeper in the specified namespace you can
 ```shell
 kubectl create -f config/samples/operator_v1alpha1_gatekeeper.yaml
 ```
+
+## Updating gatekeeper 
+
+### Update gatekeeper package according to gatekeeper version 
+
+The gatekeeper package version should be updated with the matched the gatekeeper version when gatekeeper version is updated in go.mod.
+```
+require github.com/open-policy-agent/gatekeeper/v3 v3.13.4
```

Comment:
This should be in this doc instead:
https://github.com/stolostron/gatekeeper-operator/blob/main/docs/upgrading-gatekeeper.md